sql/telemetry_derived/ssl_ratios_v1/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+friendly_name: SSL Ratios
+description: >
+  Percentages of page loads Firefox users have performed that were conducted over SSL.
+owners:
+  - chutten@mozilla.com
+labels:
```

Comment:
Needs: `incremental` and `version`

Reply:
There was some discussion in https://github.com/mozilla/bigquery-etl/pull/684 about using [`schedule:daily` instead of `incremental`](https://github.com/mozilla/bigquery-etl/pull/684#discussion_r368087046) and [the `version` label is redundant with the table name](https://github.com/mozilla/bigquery-etl/pull/684#discussion_r368081032)

Reply:
I specifically requested that we use `schedule:daily` rather than `incremental:true` because a string "true" seemed like an odd fit. But given that @scholtzan has now added support for key-less tags, it might be reasonable to call this use `incremental` as a tag rather than `schedule:daily`.

Reply:
Okay, I changed it back to the `incremental` label

Reply:
I think there's a bit of confusion here - something can be scheduled daily but not be incremental. We should probably include both.